### ISSUE TITLE
error when rolling back from a named store after modifying a variable

### ISSUE THREAD

**Issue by Elckarow** [renpy/renpy#5265]
from the 8.2 pre release
```rpy
default named.test_val = 0

screen test():
    textbutton "click" action Return()

init python in named:
    from renpy.store import Function, ui, config
    def test():
        renpy.checkpoint()
        global test_val
        test_val += 1
        renpy.invoke_in_new_context(renpy.call_screen, "test")
        test_val -= 1
        renpy.rollback(force=True, greedy=False)
    
    config.overlay_functions.append(lambda: ui.add(renpy.Keymap(K_a=Function(test))))
```

pressing `A` and clicking the button will result in the following error
```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 104, in script
    "a"
  File "game/script.rpy", line 104, in script
    "a"
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/script.rpy", line 97, in test
    renpy.rollback(force=True, greedy=False)
ValueError: not enough values to unpack (expected 2, got 1)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "PATH\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "PATH\renpy\main.py", line 635, in main
    run(restart)
  File "PATH\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "PATH\renpy\execution.py", line 955, in run_context
    context.run()
  File "game/script.rpy", line 104, in script
    "a"
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "game/script.rpy", line 104, in script
    "a"
  File "PATH\renpy\ast.py", line 2557, in execute
    Say.execute(self)
  File "PATH\renpy\ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "PATH\renpy\exports.py", line 1493, in say
    who(what, *args, **kwargs)
  File "PATH\renpy\character.py", line 1391, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "PATH\renpy\character.py", line 1042, in do_display
    display_say(who,
  File "PATH\renpy\character.py", line 738, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "PATH\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "PATH\renpy\display\core.py", line 2153, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "PATH\renpy\display\core.py", line 3183, in interact_core
    rv = root_widget.event(ev, x, y, 0)
  File "PATH\renpy\display\layout.py", line 1197, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "PATH\renpy\display\layout.py", line 1197, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "PATH\renpy\display\layout.py", line 1197, in event
    rv = i.event(ev, x - xo, y - yo, cst)
  File "PATH\renpy\display\behavior.py", line 548, in event
    rv = run(action)
  File "PATH\renpy\display\behavior.py", line 395, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 586, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/script.rpy", line 97, in test
    renpy.rollback(force=True, greedy=False)
  File "PATH\renpy\exports.py", line 1835, in rollback
    renpy.game.log.complete()
  File "PATH\renpy\rollback.py", line 614, in complete
    self.current.stores[name], self.current.delta_ebc[name] = delta
ValueError: not enough values to unpack (expected 2, got 1)

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24011405

Sun Jan 14 21:18:30 2024
```

NOT modifying `test_val` won't result in said error

commenting `renpy.checkpoint` and `renpy.rollback` and then manually rolling back will get you the error

**Comment by mal** [comment]
Direct result of `c1f906fcbafb1e9e73158e7232f876ecd0c0de8a`.

**Comment by renpytom** [comment]
Closed by cb212140765210bdf0cbd6b9c0260fb4298a0002 .

**Comment by Elckarow** [comment]
yep. working as intended now 👍
